cosmo_tester/test_suites/image_based_tests/component_test.py

```diff
@@ -0,0 +1,170 @@
+########
```

Comment:
Probably not worth putting the copyright header on, copyright is automatic anyway and we have a license file elsewhere in the repo so this is just wasting space I think?